dumpling/export/writer_test.go

```diff
@@ -146,7 +146,7 @@ func TestWriteTableData(t *testing.T) {
 	require.Equal(t, expected, string(bytes))
 
 	require.NoError(t, failpoint.Enable("github.com/pingcap/tidb/dumpling/export/FailToCloseDataFile", "return(true)"))
```

Comment:
here also `testfailpoint.Enable`. A better way to fix the defer bug.

Reply:
will also change it